ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -81,26 +83,43 @@ export class InstallationProgress extends React.Component {
 
         if (steps === undefined) { return null }
 
+        let icon;
+        let title;
+        if (status === "success") {
+            icon = CheckCircleIcon;
+            title = _("Installed");
+        } else if (status === "danger") {
+            icon = ExclamationCircleIcon;
+            title = _(_("Installation failed"));
```

Comment:
I think you want only one `_(...)` here :-)

Reply:
Fixed. thanks!